react/index.js

```diff
@@ -140,8 +140,8 @@ class ProductSummary extends Component {
         width: '100%',
         height: '100%',
       }}
-      height="100%"
-      width="100%"
+      width="100"
+      height="56"
```

Comment:
Shouldn't there be a property for maintaning proportion in smaller screens? I remember another prop in your others PR's

Reply:
Added